Add option to set window title to "spt - Spotify TUI" on startup

diegov opened the pull request:
This adds an option to set the window title to "Spotify TUI" on startup. 

This is very handy when switching windows via something like dmenu, since it makes the terminal window where spotify-tui is running easy to find. 

There are a few choices that I wasn't sure about though:

- I've added the configuration option under "behavior", is there a better place for it? 
- I've set it to enabled by default, since it's a very well supported xterm feature so it shouldn't cause problems for most people. Should it default to false to maintain existing behavior on upgrade?
- Since the binary is called spt, i've set the title to "spt - Spotify TUI", in case people want to locate the window by the name of the command they used to start the application. But just "Spotify TUI" is probably nicer and maybe locating the window by spt is not enough to justify messing with it?

Thanks!

Rigellute commented on src/main.rs:
```diff
@@ -268,7 +268,12 @@ async fn start_ui(user_config: UserConfig, app: &Arc<Mutex<App>>) -> Result<()>
+  let mut backend = CrosstermBackend::new(stdout);
+
+  if user_config.behavior.set_window_title {  
+    backend.execute(SetTitle("spt - Spotify TUI"))?;
```
Very cool!

Rigellute reviewed (approved):
Nice addition. I think placing it in the `Behavior` struct makes sense 👍.

Thanks again 👏

Rigellute commented:
I think I've fixed these clippy errors in master. But would you mind rebasing your branch?

diegov commented:
> I think I've fixed these clippy errors in master. @diegov But would you mind rebasing your branch?

I've rebased master and fixed the rustfmt issues, clippy and rustfmt are passing locally now.

Rigellute commented:
Thanks @diegov and sorry for the delay in merging this.

Rigellute commented:
@all-contributors please add @diegov for code
